fl4health/clients/basic_client.py

```diff
@@ -248,8 +241,13 @@ def process_config(self, config: Config) -> Tuple[Union[int, None], Union[int, N
         except ValueError:
             evaluate_after_fit = False
 
+        try:
+            pack_losses_with_val_metrics = narrow_dict_type(config, "pack_losses_with_val_metrics", bool)
```

Comment:
Is it worth having an INFO log indicating that val metrics and backed with losses?

Reply:
Yup. I think so. Will add it.